Kickstarter-iOS/Views/Cells/PledgeShippingLocationCell.swift

```diff
@@ -0,0 +1,106 @@
+import Library
+import Prelude
+import UIKit
+
+final class PledgeShippingLocationCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private lazy var adaptableStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var amountLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var countryButton: UIButton = { UIButton(frame: .zero) }()
+  private lazy var titleLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var spacer: UIView = {
+    UIView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  // MARK: - Lifecycle
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    _ = self
+      |> \.accessibilityElements .~ [self.titleLabel, self.countryButton, self.amountLabel]
+
+    _ = (self.rootStackView, self.contentView)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
+    self.rootStackView.addArrangedSubview(self.titleLabel)
+    self.rootStackView.addArrangedSubview(self.adaptableStackView)
+
+    self.adaptableStackView.addArrangedSubview(self.countryButton)
+    self.adaptableStackView.addArrangedSubview(self.spacer)
+    self.adaptableStackView.addArrangedSubview(self.amountLabel)
+
+    self.spacer.widthAnchor.constraint(greaterThanOrEqualToConstant: Styles.grid(3)).isActive = true
+
+    self.amountLabel.setContentCompressionResistancePriority(.required, for: .horizontal)
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> checkoutBackgroundStyle
+
+    _ = self.adaptableStackView
+      |> checkoutAdaptableStackViewStyle(self.traitCollection.ksr_isAccessibilityCategory())
+
+    _ = self.amountLabel
+      |> amountLabelStyle
+
+    _ = self.countryButton
```

Comment:
Should we consider adding a hint to this button? `"Tap to change your shipping location"`?

Reply:
I'm undecided on this one. On one hand, that button looks like a button (so it should be treated like a button - `Title, button` (a11Y-title + a11y-trait)...on the other hand we currently use a different pattern for this with more descriptive action steps `Double tap to` (which in my opinion goes slightly against HIG).

Maybe we can talk about this offline?